packages/cli/test/unit/openapi/controller-spec.unit.js

```diff
@@ -44,13 +44,13 @@ describe('openapi to controllers/models', () => {
             comments: [
               'Creates a new customer',
               '\n',
-              '@param requestBody Customer to add',
-              '@param accessToken Access token',
+              '@param _requestBody Customer to add',
+              '@param accessToken Access token (/* access_token */)',
               '@returns customer response',
             ],
             decoration: "@operation('post', '/customers')",
             signature:
-              'async createCustomer(@requestBody() requestBody: Customer, ' +
+              'async createCustomer(@requestBody() _requestBody: Customer, ' +
```

Comment:
wouldn't the name here be `customer` instead of `requestBody` or `_requestBody`

Reply:
We use parameter names from the openapi spec instead of inferring it from the type.

Reply:
I see.
Then it will always be `requestBody`, only parameters have a name in the OpenAPI spec.